pom.xml

```diff
@@ -1500,8 +1500,9 @@
                         <!-- set default options -->
                         <argLine>
                             @{jacocoArgLine}
+                            ${jdk.surefire.argLine}
                             -Xmx1500m
-                            -XX:MaxDirectMemorySize=512m
+                            -XX:MaxDirectMemorySize=2500m
```

Comment:
Because one of the configs that I removed had it set to that value and, well, this setting is meaningless anyway (it has absolutely no functional impact on the process, all it does it set a threshold for when exceptions get thrown)